gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 # Commons
-kotlin = "1.8.10"
-kotlinSP = "1.8.10-1.0.9"
+kotlin = "1.9.10"
```

Comment:
why do we need this change?

More over, we cannot use Kotlin 1.8.20 and above because of the following issue https://youtrack.jetbrains.com/issue/KT-60748 - we either need to apply workaround provided, or rename tests.